editor/WorldTexturesDialog.cpp

```diff
@@ -2318,7 +2320,8 @@ void CWorldTexturesDialog::OnSaveAsTga()
 	if (! strchr(filename,'.'))
 		strcat(filename,".tga");
 
-	bm_SaveBitmapTGA (filename,GameTextures[n].bm_handle);
+	// FIXME: Save as PNG
+	// bm_SaveBitmapTGA (filename,GameTextures[n].bm_handle);
```

Comment:
This can probably use the code I added for PNG support, although not sure if you want to do that on this PR

Reply:
in another PR, sure :)